bower.json

```diff
@@ -32,7 +32,7 @@
     "lodash": "^3.7.0",
     "mocha": "^3.1.2",
     "sinon-chai": "^2.7.0",
-    "sinonjs": "^1.14.1",
+    "sinonjs": "http://sinonjs.org/releases/sinon-1.17.1.js",
```

Comment:
We (the Sinon maintainers) would much prefer if you would use the releases from the CDN, than from the documentation site (which is graciously hosted by GitHub).

```
https://cdn.jsdelivr.net/npm/sinon@1.17.1/pkg/sinon.js
```